concepts/state-management/README.md

```diff
@@ -1,6 +1,14 @@
 # State management
 
-Dapr makes it simple for you to store key/value data in a store of your choice.
+Dapr comes with APIs for a key/value storage system, called "state management" in Dapr terminology. If your microservice uses state management, it automatically gets access to any of the [supported databases](https://github.com/dapr/docs/blob/master/howto/setup-state-store/supported-state-stores.md), without adding or learning a third party SDK.
```

Comment:
supported state stores is a better phrasing than supported databases.
We don't want to confuse readers with databases since a state store is different (Dapr does not support queries and other DB features)